docs/build/test-and-debug/foundry.md

```diff
@@ -11,18 +11,18 @@ For instructions on how to install `foundry-zksync` please refer to the [Getting
 
 ## Overview
 
-`foundry-zksync`, a fork of Foundry, provides developers with a tailored testing framework designed specifically for zkSync environments. Utilizing `zkforge test`, you can execute your smart contract tests efficiently. Tests are written in Solidity, and the framework is designed to recognize any contract function prefixed with `test` as a test case. By convention, tests are typically stored within the `test/` directory and have a `.t.sol` extension.
+`foundry-zksync`, a fork of Foundry, provides developers with a tailored testing framework designed specifically for zkSync environments. Utilizing `forge test --zksync`, you can execute your smart contract tests efficiently. Tests are written in Solidity, and the framework is designed to recognize any contract function prefixed with `test` as a test case. By convention, tests are typically stored within the `test/` directory and have a `.t.sol` extension.
 
 :::info
 For more detailed documentation related to Foundry testing please refer to the official upstream Foundry documentation [here](https://book.getfoundry.sh/forge/tests).
 :::
 
 ## Running Tests
 
-To initiate your tests, use the `zkforge test` command. This command automatically locates and executes tests across your source directory. Here's an example of executing tests in a standard project setup:
+To initiate your tests, use the `forge test --zksync` command with the `--zksync` flag or make us of `vm.zkVm(true)` in your tests. This command automatically locates and executes tests across your source directory. Here's an example of executing tests in a standard project setup:
```

Comment:
"make *use* of".  Also seems redundant with specifying the --zksync flag after declaring the command with the flag?